src/teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './add'
+
+class Teacher < Person
+  def initialization(specialization)
```

Comment:
- [ ] I think here you missed some parameters so here when inheriting a class so also it inherit the paramters so please add the parameters for age, name and parent_permission with their defult values 💯🔥💪🏻😊

Reply:
@Reem-lab Done, thanks.

Reply:
- [ OPTIONAL ] I think you missunderstand me so it could be somthing like this

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```